test/e2e/event_transformation_test.go

```diff
@@ -116,7 +117,12 @@ func TestEventTransformation(t *testing.T) {
 	// check if the logging service receives the correct number of event messages
 	expectedContent := body + msgPostfix
 	expectedContentCount := len(subscriptionNames1) * len(subscriptionNames2)
-	if err := WaitForLogContentCount(clients, loggerPod.Name, loggerPod.Spec.Containers[0].Name, ns, expectedContent, expectedContentCount); err != nil {
+	podName := loggerPod.Name
+	containerName := loggerPod.Spec.Containers[0].Name
+	if err := WaitForLogContentCount(clients, podName, containerName, ns, expectedContent, expectedContentCount); err != nil {
+		if logs, err := clients.Kube.PodLogs(podName, containerName, ns); err != nil {
```

Comment:
Is this right? If you get an error when calling PodLogs, you're dumping the contents of the logs (are they even valid??) but ignoring the error returned. Otherwise, if you get the logs, you're not dumping them.

Reply:
Sorry my mistake. It should be `err == nil`.